scripts/deploy

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+create_cert() {
+  cat <<EOF > public.crt
+-----BEGIN CERTIFICATE-----
+MIICSTCCAe+gAwIBAgIQWgUVWCiZdyOGruNe6m4iWjAKBggqhkjOPQQDAjBMMRww
+GgYDVQQKExNDZXJ0Z2VuIERldmVsb3BtZW50MSwwKgYDVQQLDCNlbGl5YW1sZXZ5
+QEVsaXlhbXMtTUJQLmF0dGxvY2FsLm5ldDAeFw0yMjA1MTExNDAxMjBaFw0zMjA1
+MTEwMjAxMjBaMEwxHDAaBgNVBAoTE0NlcnRnZW4gRGV2ZWxvcG1lbnQxLDAqBgNV
+BAsMI2VsaXlhbWxldnlARWxpeWFtcy1NQlAuYXR0bG9jYWwubmV0MFkwEwYHKoZI
+zj0CAQYIKoZIzj0DAQcDQgAEFH8UPCl/vAHkMbTF3E8yhSdLNH2XueKUHns+O4FR
+hn096OJKnGZFb/HiW9iJWhj4CJ4LubSvsiZJZ7YuDlM9faOBsjCBrzAOBgNVHQ8B
+Af8EBAMCAqQwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDwYDVR0TAQH/BAUwAwEB/zAd
+BgNVHQ4EFgQUImjresql78fBpwSV7lp4fT4+NnwwWAYDVR0RBFEwT4IFbWluaW+C
+C21pbmlvLm1pbmlvgg9taW5pby5taW5pby5zdmOCHW1pbmlvLm1pbmlvLnN2Yy5j
+bHVzdGVyLmxvY2Fsgglsb2NhbGhvc3QwCgYIKoZIzj0EAwIDSAAwRQIgWT4CU5ib
+LNeXJmh2lnqEvaeKgqLHPFgMOQg+4TyO+uQCIQCI5WX1E84B+z6yX7WKIBYJIjto
+RjQi75QniF10pi2jKA==
+-----END CERTIFICATE-----
+EOF
+
+  cat <<EOF > private.key
+-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgRiROiBUTvzxfDbiu
+60w9WdlRKAI7Jy0c26uC2FoAZdahRANCAAQUfxQ8KX+8AeQxtMXcTzKFJ0s0fZe5
+4pQeez47gVGGfT3o4kqcZkVv8eJb2IlaGPgIngu5tK+yJklnti4OUz19
+-----END PRIVATE KEY-----
+EOF
+}
+
+deploy_minio() {
+  KUBECTL_CMD="kubectl"
+  K3S_FLAG=0
+  if command -v k3s &> /dev/null ; then
+      KUBECTL_CMD="k3s kubectl"
+      K3S_FLAG=1
+  fi
+
+  if [ -z "${KUBECONFIG}" ]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 1
+  fi
+
+  helm repo add minio https://charts.min.io/
+  helm repo update
+
+  echo "Creating Minio Namespace"
+  ${KUBECTL_CMD} create ns minio
+
+  echo "Creating Minio tls-ssl secret"
+  ${KUBECTL_CMD} -n minio create secret generic tls-ssl-minio --from-file=private.key --from-file=public.crt
+
+  echo "Installing Minio"
+  helm install --namespace minio --set rootUser=inspectorgadget,rootPassword=gogadgetgo --set tls.enabled=true,tls.certSecret=tls-ssl-minio --set replicas=1 --set resources.requests.memory=2Gi --set persistence.enabled=false --set drivesPerNode=0 --set pools=0 --set mode=standalone --set buckets[0].name=rancherbackups --set buckets[0].policy=none --set buckets[0].purge=true minio minio/minio
+  
+  if [ ${K3S_FLAG} -eq 1 ] ; then
+      time timeout 300 bash -c 'while ! (k3s kubectl --namespace minio rollout status --timeout 10s deploy/minio 2>/dev/null); do sleep 5; done'
+  fi
+
+  ${KUBECTL_CMD} create secret generic miniocreds --from-literal=accessKey=inspectorgadget --from-literal=secretKey=gogadgetgo
+}
+
+deploy_backup_restore() {
+  if [ -z "${KUBECONFIG}" ]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+
+  . ./scripts/version &> /dev/null
+
+  helm repo add rancher-chart https://charts.rancher.io
```

Comment:
this seems unused?

Reply:
What does?

Reply:
`rancher-chart` repo is unused

Reply:
ahh, yea i think that got left over accidentally. nice catch

Reply:
That was from back when I was using the actual rancher rancher at first to test